### ISSUE TITLE
Switcher_aio inputs error

### ISSUE THREAD

**Issue by Ardeth** [TomerFi/home-assistant-custom-components#31]
Hi,

trying to select inputs from the drop down selection give the next error - 
**failed to call service input_select/select_option**
so if I choose for example 30 mins timer its still turn it on to 15 mins timer.
same for the auto off and shudeules.

**Comment by TomerFi** [comment]
Yeah it's a common problem that will be resolved in the new component I'm currently working on.
Please take a look [here](https://github.com/TomerFi/home-assistant-custom-components/issues/25#issuecomment-452028215). While the issue in the link I provided talks about `input_text` component, the same applies for the `input_select` component to.
Just configure a dummy [Input Select](https://www.home-assistant.io/components/input_select/) entity and that should fix the issue.
